x-pack/plugins/cloud_security_posture/common/schemas/csp_rule.ts

```diff
@@ -6,8 +6,6 @@
  */
 import { schema as rt, TypeOf } from '@kbn/config-schema';
 
-export const cspRuleAssetSavedObjectType = 'csp_rule';
```

Comment:
Moved to `constants` because importing this constant from here was also importing in the bundle `@kbn/config-schema`.